styles/modules/Policy.module.scss

```diff
@@ -177,9 +187,18 @@
 }
 
 .confirmDeleteText {
+  text-align: center;
   margin-bottom: 1.5rem;
 }
 
+.updateMessageText {
+  text-align: center;
```

Comment:
I think I prefer this left-aligned, but I could go either way:

![Screen Shot 2021-06-22 at 10 17 29 AM](https://user-images.githubusercontent.com/9082799/122941099-1267e380-d343-11eb-958b-ba005409e6f1.png)